cumulus/pallets/aura-ext/src/lib.rs

```diff
@@ -70,20 +74,7 @@ pub mod pallet {
 			// Fetch the authorities once to get them into the storage proof of the PoV.
 			Authorities::<T>::get();
 
-			let new_slot = pallet_aura::CurrentSlot::<T>::get();
```

Comment:
it feels a bit weird that the pallet now has a `RelaySlotInfo` storage item that is never updated by the pallet (but only in an optional `ConsensusHook`, which parachains could in theory overwrite). Not a big deal though